lib/src/kbasesearchengine/events/handler/WorkspaceEventHandler.java

```diff
@@ -570,4 +573,108 @@ private static String toWSRefPath(final List<GUID> objectRefPath) {
         return String.join(";", refpath);
     }
 
+    @Override
+    public StatusEvent updateEvent(
```

Comment:
I think the arg can be on the same line right, since it's < 100 chars.

Arg can be final as well

Reply:
Fixed.